src/lbaf/Applications/rank_object_enumerator.py

```diff
@@ -266,7 +266,7 @@ def get_conf() -> dict:
     CONF = get_conf()
 
     # Define number of ranks
-    N_RANKS = CONF.get("x_procs") * CONF.get("y_procs") * CONF.get("z_procs")
+    N_RANKS = CONF.get("x_ranks") * CONF.get("y_ranks") * CONF.get("z_ranks")
```

Comment:
This doesn't matter. As spoken rom should be removed and `x_ranks`, `y_ranks` and `z_ranks` are now under `generate_meshes` key in configuration, so it will not work anyway, because the returned vaule of `ranks` will be None.

Reply:
`procs` is older terminology. We now use `ranks` instead everywhere. Those `_procs` were leftover from initial implementations where the notion of "processor" was used.

Reply:
I am not saying, it's not right, just as we agreed, that roe will be removed, so I see no point of editing roe anymore. If I remember well, one or more functions should stay and the rest should be removed, right?